src/main/java/Duke.java

```diff
@@ -1,10 +1,18 @@
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private static TaskList tasklist;
+
+    public static void main(String[] args) throws DukeException {
+        Task[] tasks = new Task[100];
+        tasklist = new TaskList(tasks);
+
+        Storage.readSavedList(tasklist);
+
+        Ui.greet();
+
+        String input = Ui.readCommand();
+
+        Parser.Parse(input, tasklist);
+
+        Ui.exit();
     }
-}
+}
```

Comment:
I like the simple flow of your whole process. Do consider hiding out these details into a private method and just Duke.run() or something in your main method.